pkg/common/cns-lib/vsphere/virtualcenter.go

```diff
@@ -99,22 +99,25 @@ type VirtualCenterConfig struct {
 	// Specifies whether to verify the server's certificate chain. Set to true to
 	// skip verification.
 	Insecure bool
-	// Specifies the path to a CA certificate in PEM format. This has no effect if
-	// Insecure is enabled. Optional; if not configured, the system's CA
+	// Specifies the path to a CA certificate in PEM format. This has no effect
+	// if Insecure is enabled. Optional; if not configured, the system's CA
 	// certificates will be used.
 	CAFile string
-	// Thumbprint specifies the certificate thumbprint to use
-	// This has no effect if InsecureFlag is enabled.
+	// Thumbprint specifies the certificate thumbprint to use. This has no effect
+	// if InsecureFlag is enabled.
 	Thumbprint string
-	// RoundTripperCount is the SOAP round tripper count. (retries = RoundTripperCount - 1)
+	// RoundTripperCount is the SOAP round tripper count.
+	// retries = RoundTripperCount - 1
 	RoundTripperCount int
 	// DatacenterPaths represents paths of datacenters on the virtual center.
 	DatacenterPaths []string
-	// TargetDatastoreUrlsForFile represents URLs of file service enabled vSAN datastores in the virtual center.
+	// TargetDatastoreUrlsForFile represents URLs of file service enabled vSAN
```

Comment:
Not related to your change, but this should be `TargetvSANFileShareDatastoreURLs represents ...`

Reply:
Fixed